internal/services/engines/swift/rules.go

```diff
@@ -23,22 +23,25 @@ import (
 	"github.com/ZupIT/horusec-engine/text"
 )
 
-func NewSQLiteDatabase() text.TextRule {
-	return text.TextRule{
-		Metadata: engine.Metadata{
-			ID:          "HS-SWIFT-1",
-			Name:        "SQLite Database",
-			Description: "App uses SQLite Database. Sensitive Information should be encrypted.",
-			Severity:    severities.Medium.ToString(),
-			Confidence:  confidence.Low.ToString(),
-		},
-		Type: text.AndMatch,
-		Expressions: []*regexp.Regexp{
-			regexp.MustCompile(`sqlite3_exec`),
-			regexp.MustCompile(`sqlite3_finalize`),
-		},
-	}
-}
+// DEPRECATED This rule is not usage really in any swift project,
+// because when use sqlite3_exec internally it's running the commands sqlite3_prepare_v2, sqlite3_step, sqlite3_finalize
+// then is not necessary use sqlite3_finalize and this rule not will get anywhere vulnerability
+//func NewSQLiteDatabase() text.TextRule {
+//	return text.TextRule{
+//		Metadata: engine.Metadata{
+//			ID:          "HS-SWIFT-1",
+//			Name:        "SQLite Database",
+//			Description: "App uses SQLite Database. Sensitive Information should be encrypted.",
+//			Severity:    severities.Medium.ToString(),
+//			Confidence:  confidence.Low.ToString(),
+//		},
+//		Type: text.AndMatch,
+//		Expressions: []*regexp.Regexp{
+//			regexp.MustCompile(`sqlite3_exec`),
+//			regexp.MustCompile(`sqlite3_finalize`),
+//		},
+//	}
+//}
```

Comment:
I was thinking, wouldn't be better let the function and just put a `// Deprected: ...` on godoc? What do you think? I don't know if comment the entire function is a good strategy, seems a dead code that should be removed (but we can't)

 
```go
// Deprecated: This rule is not usage really in any swift project,
// because when use sqlite3_exec internally it's running the commands sqlite3_prepare_v2, sqlite3_step, sqlite3_finalize
// then is not necessary use sqlite3_finalize and this rule not will get anywhere vulnerability
func NewSQLiteDatabase() text.TextRule {
	return text.TextRule{
		Metadata: engine.Metadata{
			ID:          "HS-SWIFT-1",
			Name:        "SQLite Database",
			Description: "App uses SQLite Database. Sensitive Information should be encrypted.",
			Severity:    severities.Medium.ToString(),
			Confidence:  confidence.Low.ToString(),
		},
		Type: text.AndMatch,
		Expressions: []*regexp.Regexp{
			regexp.MustCompile(`sqlite3_exec`),
			regexp.MustCompile(`sqlite3_finalize`),
		},
	}
}
```

Reply:
![image](https://user-images.githubusercontent.com/63816070/143439756-f458e1df-1f6d-40c3-8d4b-88771f65e681.png)
This format is being used in others places of the project, is necessary changes they're too ?

Reply:
I think that would be good, for consistency.

Reply:
done :)

Reply:
I think that I wasn't clear here, but I was thinking that would be good remove the comment from entire function, so for example, instead of 
```go
// Deprecated: Repeated vulnerability, same as HS-JAVA-5
//
//func NewXMLParsingVulnerableToXXEWithSchemaFactory() text.TextRule {
//	return text.TextRule{
//		Metadata: engine.Metadata{
//			ID:          "HS-JAVA-6",
//			Name:        "XML parsing vulnerable to XXE With TransformerFactory",
//			Description: "XML External Entity (XXE) attacks can occur when an XML parser supports XML entities while processing XML received from an untrusted source. For more information checkout the CWE-611 (https://cwe.mitre.org/data/definitions/611.html) advisory.",
//			Severity:    severities.Medium.ToString(),
//			Confidence:  confidence.Low.ToString(),
//		},
//		Type: text.AndMatch,
//		Expressions: []*regexp.Regexp{
//			regexp.MustCompile(`SchemaFactory\.newInstance\(`),
//		},
//	}
//}
```
Use:
```go
// Deprecated: Repeated vulnerability, same as HS-JAVA-5
//
func NewXMLParsingVulnerableToXXEWithSchemaFactory() text.TextRule {
	return text.TextRule{
		Metadata: engine.Metadata{
			ID:          "HS-JAVA-6",
			Name:        "XML parsing vulnerable to XXE With TransformerFactory",
			Description: "XML External Entity (XXE) attacks can occur when an XML parser supports XML entities while processing XML received from an untrusted source. For more information checkout the CWE-611 (https://cwe.mitre.org/data/definitions/611.html) advisory.",
			Severity:    severities.Medium.ToString(),
			Confidence:  confidence.Low.ToString(),
		},
		Type: text.AndMatch,
		Expressions: []*regexp.Regexp{
			regexp.MustCompile(`SchemaFactory\.newInstance\(`),
		},
	}
}
```

What do you think?

Reply:
then the rule HS-JAVA-6 is correctly

Reply:
This is what I saying. Remove the comments for all rules that is not used and just set the `// Deprecated` comment.

I'm just proposing this change because I think commented code is dead code that should be removed from the repository, but we can't actually remove these code.

Reply:
done